db.js

```diff
@@ -0,0 +1,257 @@
+  {   "ID": 55,   "Mountain Peak": "El Diente Peak",   "Mountain Range": "San Juan Mountains",   "RangeID": 4,   "Elevation_ft": 14159,   "fourteener": "N",   "Prominence_ft": 239,   "Isolation_mi": 0.75,   "Lat": 37.839383,   "Long": -108.005335,   "Standard Route": "North Slopes ",   "Distance_mi": 15,   "Elevation Gain_ft": 5000,   "DifficultyDescription": "Class 3",   "Difficulty": 3,   "Traffic Low": 1000,   "Traffic High": 3000,   "photo": "https://www.14ers.com/photos/wilsongroup/peakphotos/large/201007_ElD01.jpg" },
+  {   "ID": 56,   "Mountain Peak": "Conundrum Peak",   "Mountain Range": "Elk Mountains",   "RangeID": 1,   "Elevation_ft": 14060,   "fourteener": "N",   "Prominence_ft": 280,   "Isolation_mi": 0.4,   "Lat": 39.015682,   "Long": -106.862749,   "Standard Route": "Via Castle Peak ",   "Distance_mi": 14.5,   "Elevation Gain_ft": 4850,   "DifficultyDescription": "Hard Class 2",   "Difficulty": 2,   "Traffic Low": 1000,   "Traffic High": 3000,   "photo": "https://www.14ers.com/photos/castlegroup/peakphotos/large/200807_Cast01.jpg" },
+  {   "ID": 57,   "Mountain Peak": "North Eolus",   "Mountain Range": "San Juan Mountains",   "RangeID": 4,   "Elevation_ft": 14039,   "fourteener": "N",   "Prominence_ft": 199,   "Isolation_mi": 0.25,   "Lat": 37.625192,   "Long": -107.621187,   "Standard Route": "South Ridge ",   "Distance_mi": 16.75,   "Elevation Gain_ft": 6000,   "DifficultyDescription": "Class 3",   "Difficulty": 3,   "Traffic Low": 3000,   "Traffic High": 5000,   "photo": "https://www.14ers.com/photos/windomgroup/peakphotos/large/200607_Eolus01.jpg" },
+  {   "ID": 58,   "Mountain Peak": "North Maroon Peak",   "Mountain Range": "Elk Mountains",   "RangeID": 1,   "Elevation_ft": 14014,   "fourteener": "N",   "Prominence_ft": 254,   "Isolation_mi": 0.37,   "Lat": 39.076007,   "Long": -106.987058,   "Standard Route": "Northeast Ridge ",   "Distance_mi": 9.25,   "Elevation Gain_ft": 4500,   "DifficultyDescription": "Class 4",   "Difficulty": 4,   "Traffic Low": 1000,   "Traffic High": 3000,   "photo": "https://www.14ers.com/photos/maroongroup/peakphotos/large/200807_NMar01.jpg" }
+]
+
+const rangeData = [
+  {
+    ID: 1,
+    Name: 'Elk Mountains'
+  },
+  {
+    ID: 2,
+    Name: 'Front Range'
+  },
+  {
+    ID: 3,
+    Name: 'Mosquito Range'
+  },
+  {
+    ID: 4,
+    Name: 'San Juan Mountains'
+  },
+  {
+    ID: 5,
+    Name: 'Sangre de Cristo Range'
+  },
+  {
+    ID: 6,
+    Name: 'Sawatch Range'
+  }
+]
+
+const conn = new Sequelize(process.env.DATABASE_URL || 'postgres://localhost/14er_ranges_peaks_db');
+
+const syncAndSeed = async () => {
+  await conn.sync({ force: true });
+  await Promise.all(data.map(mountain => Mountain.create(mountain)));
+  await Promise.all(rangeData.map(range => Range.create(range)));
```

Comment:
fun fact you can do this 
```suggestion
  await Promise.all(data.map(Mountain.create));
  await Promise.all(rangeData.map(Range.create));
```

i honestly dont know how to explain it in words but im sure you can see the pattern-
```js
 await Promise.all(data.map(mountain => Mountain.create(mountain)));
  await Promise.all(rangeData.map(range => Range.create(range)));
vs
 await Promise.all(data.map(Mountain.create));
  await Promise.all(rangeData.map(Range.create));

```